ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/link-tracking-groups.j2

```diff
@@ -0,0 +1,21 @@
+{# Link Traking Groups #}
+{% if link_tracking_groups is arista.avd.defined %}
+
+## Link Tracking
+
+### Link Tracking Groups Summary
+
+| Group Name | Minimum Links | Recovery Delay |
+| ---------- | ------------- | -------------- |
+{%     for link_tracking_group in link_tracking_groups | arista.avd.natural_sort('name') %}
+{%         if link_tracking_group.name is arista.avd.defined %}
+| {{ link_tracking_group.name }} | {{ link_tracking_group.links_minimum | arista.avd.default("default") }} | {{ link_tracking_group.recovery_delay | arista.avd.default("default") }} |
```

Comment:
default value should be `-` like we have elsewhere.

Reply:
ok